test/Airgrab.test.ts

```diff
@@ -53,6 +48,14 @@ describe('Airgrab', function () {
       throw new Error('Chain ID not found');
     }
 
+    // if chain is celo or baklava skip tests
```

Comment:
How about we explain the reason why we are skipping these networks or just remove the comment altogether?